tests/ci_tests/code_formatter/conftest.py

```diff
@@ -0,0 +1,28 @@
+"""Fixtures and configuration for code_formatter tests"""
```

Comment:
Added a docstring as specified in the style guide.
```suggestion
"Fixtures and configuration for code_formatter tests"
```